src/components/GraphiQL.js

```diff
@@ -694,6 +710,42 @@ export class GraphiQL extends React.Component {
     editor.setValue(print(parse(editor.getValue())));
   };
 
+  handleDefragQuery = () => {
+    const editor = this.getQueryEditor();
+    const ast = parse(editor.getValue());
+
+    let operations = ast.definitions.filter(elem => {
+      return elem.kind !== 'FragmentDefinition';
```

Comment:
nit:
```
const operations = ast.definitions.filter(elem => elem.kind !== 'FragmentDefinition');
```

Reply:
Actually, we don't need to compute operations here at all - we could just filter at line 730 where we do actually need the operations. This is unnecessary